packages/node/src/integrations/localvariables.ts

```diff
@@ -281,7 +282,11 @@ export class LocalVariables implements Integration {
     addGlobalEventProcessor: (callback: EventProcessor) => void,
     clientOptions: NodeClientOptions | undefined,
   ): void {
-    if (this._session && clientOptions?.includeLocalVariables) {
+    // Only setup this integration if the Node version is >= v18
+    // https://github.com/getsentry/sentry-javascript/issues/7697
+    const supportedNodeVersion = (NODE_VERSION.major || 0) >= 18;
```

Comment:
Let's log out a debug log if they try to use this without a supported Node version.